golang/server/server.go

```diff
@@ -172,3 +205,16 @@ func (server MinimalGRPCServer) loadTlsCredentials() credentials.TransportCreden
 	}
 	return credentials.NewTLS(tlsConfig)
 }
+
+func isGracefulExit(err error) bool {
```

Comment:
This is interesting bit; in order to do a graceful I need to ignore the errors seen in this method. For more information look at the following:

https://github.com/jaegertracing/jaeger/issues/3025#issuecomment-847517715
https://github.com/jaegertracing/jaeger/blob/main/cmd/query/app/server.go

Reply:
Can we paste that comment link as a comment here